global-script.lua

```diff
@@ -2403,9 +2376,9 @@ function MapPlaceCustom()
     local maps = getMapTiles()
     -- board type is guaranteed to either be thematic or normal, and there has to be at least one map tile in the table
     if string.sub(maps[1].getName(),1,4) == "NORM" then
-        alternateSetupIndex = 1
+        boardLayout = "Custom"
```

Comment:
what's the idea behind this being `Custom`?

Reply:
This is not one the the standard layouts, so I record that. I'd use that for both normal and  thematic, except things use `Thematic` for calculating difficulty.